lib/setup.js

```diff
@@ -123,6 +123,22 @@ let objects; // instance
 let States;  // constructor
 let states;  // instance
 
+/** Prints the config file to the console */
```

Comment:
This was moved to here:
https://github.com/ioBroker/ioBroker.js-controller/blob/c6854610b905de782cee49a2eabc1cb2ed18ba17/lib/cli/cliProcess.js#L215
Where is it used from? Maybe it should be in cliTools instead?

Reply:
The github search shows no references to this method...

Reply:
yes but not in 1.5.x branch!! in 1.5.x this do not exists, or?

Reply:
in 2.x branch all is ok I think, but in 1.5.x branch it was makred as error

Reply:
You are right, I forgot this PR is for 1.5.x

Reply:
... I could seacrh all old 1.5.x commits where it was made wrong, but decided just do it it that way for 1.5.x